app/src/main/java/org/oppia/android/app/splash/SplashActivityPresenter.kt

```diff
@@ -67,6 +78,14 @@ class SplashActivityPresenter @Inject constructor(
     subscribeToOnboardingFlow()
   }
 
+  fun handleOnDeprecationNoticeActionClicked(noticeType: DeprecationNoticeActionType) {
+    when (noticeType) {
+      DeprecationNoticeActionType.CLOSE -> handleOnDeprecationNoticeCloseAppButtonClicked()
```

Comment:
I have both so that the old flow still works. I had removed the older `DeprecationNoticeExitAppListener` and Ben was of the idea that I should keep both until the feature is stable and then we can remove the older listener and all the code related to it. In the meantime, we should keep both implementations and only gate the feature.